### PR TITLE
[DOCFIX] Update security docs

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

**Comment by gpang** [comment]
@calvinjia

File: docs/en/advanced/Security.md [hunk 1]
```diff
@@ -100,23 +98,30 @@ The output of the `ls` shell command when authorization is enabled is:
...
+This implementation executes the `groups` shell command on the local machine
+to fetch the group memberships of a particular user.
+Running the `groups` command for every query may be expensive, so
+the user group mapping is cached, with an expiration period configured by the
 `alluxio.security.group.mapping.cache.timeout` property, with a default value of `60s`.
-If set to a value of `0`, caching is disabled.
+If set to a value of `0`, the caching is disabled.
+If the cache timeout is too low or disable, the `groups` command will be run very frequently, but
+may increase latency for operations.
+If the cache timeout is too higg, the `groups` command will not be run frequently, but the cached
```
Comment (ns1123) [inline docs/en/advanced/Security.md]: NIT 'high'

File: docs/en/advanced/Security.md [hunk 2]
```diff
@@ -100,23 +98,30 @@ The output of the `ls` shell command when authorization is enabled is:
...
-This implementation executes the `groups` shell command to fetch the group memberships of the given user.
-The user group mapping is cached, with an expiration period configured by the
+This implementation executes the `groups` shell command on the local machine
+to fetch the group memberships of a particular user.
+Running the `groups` command for every query may be expensive, so
+the user group mapping is cached, with an expiration period configured by the
 `alluxio.security.group.mapping.cache.timeout` property, with a default value of `60s`.
-If set to a value of `0`, caching is disabled.
+If set to a value of `0`, the caching is disabled.
+If the cache timeout is too low or disable, the `groups` command will be run very frequently, but
```
Comment (ns1123) [inline docs/en/advanced/Security.md]: NIT 'disabled'

File: docs/en/advanced/Security.md [hunk 3]
```diff
@@ -349,5 +360,5 @@ respective under file system, such as HDFS transparent encryption or Linux disk
 
 ## Deployment
 
-It is recommended to start Alluxio master and workers under the same user.
+It is recommended to start Alluxio master and workers using the same operating system user.
```
Comment (ns1123) [inline docs/en/advanced/Security.md]: Can we just enforce this as a requirement? Many scripts like 'alluxio-start.sh all' won't work if the OS user differs. I believe our SecondaryMaster health check also relies on SSH and can't specify the OS user of the secondaries.

**Comment by calvinjia** [comment]
Merged in #9340

**Pull request closed without merging**